index.js

```diff
@@ -22,6 +22,11 @@ module.exports = robot => {
   async function handler(context) {
     const release = context.payload.release;
 
+    // Don't generate release notes for prereleases
```

Comment:
I still feel like release notes are handy for pre-releases, no?

Reply:
I was thinking this would be confusing if changes are listed multiple times

Reply:
Ah nevermind, thought this was referring to prerelease PRs, not the Github releases page. Yeah makes sense to leave out the notes for prereleases